### ISSUE TITLE
Track.description() is implemented incorrectly

### ISSUE THREAD

**Issue by SE2Dev** [lerouxrgd/datachannel-rs#26]
https://github.com/lerouxrgd/datachannel-rs/commit/f68f576e99d4e9089c9937dc0fb26cf932fc4b8d#diff-2115887ac7839a47ef00ba2ccfdf694540334556b18acc69615283f49c0edf70R222

In `libdatachannel` the track description only contains a small subsection of the overall session description; in the case of `datachannel-rs`, using `parse_media` will fail in all cases. The correct implementation of `Track.description()` would do something like this:
```rs
// ...
let description; // track description from libdatachannel
let mut lines: Vec<SdpLine> = description
    .split("\n")
    .enumerate()
    .map(|(line_number, line)| parse_sdp_line(line, line_number).expect("unable to parse line"))
    .collect();
let media = parse_media_vector(&mut lines).expect("can't parse media vector");
// ...
```

This is currently blocked by https://github.com/mozilla/webrtc-sdp/pull/262 because `parse_sdp_line` is not public.

**Comment by lerouxrgd** [comment]
Waiting on [webrtc-sdp issue](https://github.com/mozilla/webrtc-sdp/issues/265)

**Comment by lerouxrgd** [comment]
@SE2Dev 
I have released a fix for this now that webrtc-sdp 0.3.10 has finally been published to crates.io
Could you give it a try ?

**Comment by lerouxrgd** [comment]
I have released it in `0.10.0`, don't hesitate to re-open if something is wrong.
